broker-daemon/state-machines/plugins/abstract.js

```diff
@@ -31,7 +31,7 @@ class StateMachineAbstractPlugin {
    * State machine plugins expose a `configure` method that gets called against the config object when building the factory
    * By default, we use it to apply our custom transitions
    * @see https://github.com/jakesgordon/javascript-state-machine/blob/master/src/config.js#L119
-   * @param {StateMachine~Config} config - State machine configuration object
```

Comment:
no way to do a typedef based on the external lib, or not worth it?

Reply:
I didn't see anything exported by the library that would be useful for this